docs/docsite/rst/user_guide/playbooks_tags.rst

```diff
@@ -154,9 +154,9 @@ Playbook file::
 
     - hosts: all
       tasks:
-      - include_role:
-          name: myrole
-        tags: mytag
+        - include_role:
```

Comment:
Something we have to discuss in the Documentation WG to have consistent examples.

My preference is to ident minimally (so only the amount required by YAML).
Just like I recommend quoting only when it is necessary.
This gives users a better feeling to how YAML parses files and when something is mandatory.

Reply:
+1

Reply:
+1 to fixing indentation, though I don't think it's our highest priority